src/helpers/math.js

```diff
@@ -0,0 +1,153 @@
+import {isArray} from '../util/utils.js';
+
+/**
+ * Determine whether or not two values are equal (===).
+ * Example usage:
+ *      {{eq '3' 3}}    => false
+ *
+ * @param value1
+ * @param value2
+ * @returns boolean
+ */
+function eq(value1, value2) {
```

Comment:
How about having something like `logic.js`. Because those helpers have absolutely nothing to do with math but still it says math.js, kind of weird :smile: